UI/app.tsx

```diff
@@ -432,6 +434,21 @@ export function AppComponent(props: {
         }
     }
 
+    let popover: h.JSX.Element | undefined;
+    if (model.dm.search.isSearching) {
```

Comment:
In the next PR that addresses https://github.com/BlowaterNostr/blowater/issues/90, please make the popover component able to close itself by itself instead of by parent component so that the show/hide of popover is not controlled by `isSearching`.

`isSearching` should only control what's inside the popover container.